improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -78,47 +77,49 @@ class Test_run(IrisTest):
 
     RADIUS = 2500
 
+    def setUp(self):
+        """Create test cubes"""
+
+        # Create 2D test cube
+
+        data = np.ones((5, 5), dtype=np.float32)
+        data[2, 2] = 0
+        self.cube = set_up_variable_cube(data, spatial_grid="equalarea",)
+
+        # Create 3D test cube with 2 realizations
+
+        data = np.ones((2, 5, 5), dtype=np.float32)
+        self.multi_cube = set_up_variable_cube(data, spatial_grid="equalarea",)
```

Comment:
Minor quibble but I'd call this `self.multi_realization_cube` - it's a bit more descriptive.

Reply:
Done.